src/lib/useSyncPoints.js

```diff
@@ -0,0 +1,24 @@
+import { useEffect } from 'react';
+import { isEqual } from 'lodash';
+
+import { usePointCache } from 'store/pointCache';
+import usePreviousValue from 'lib/usePreviousValue';
+
+// sync a known point to the cache (i.e display values)
+export function useSyncKnownPoints(points = []) {
+  //
```

Comment:
TODO?

Reply:
right now there are no values we need to display a point card beyond their actual point.

in the future we'll need the info to display "key type" in the list of points